suitcase/mongo_normalized/__init__.py

```diff
@@ -90,6 +92,24 @@ def __call__(self, name, doc):
         sanitized_doc = event_model.sanitize_doc(doc)
         return super().__call__(name, sanitized_doc)
 
+    def update(self, name, doc):
+        if name in ['start', 'stop']:
+            current_col = getattr(self, f'_run_{name}_collection')
+            revisions_col = getattr(self, f'_run_{name}_collection_revisions')
+            old = current_col.find_one({'uid': doc['uid']})
+            old.pop('_id')
+            target_uid_docs = revisions_col.find({'uid': doc['uid']})
+            cur = target_uid_docs.sort([('revision', -1)]).limit(1)
```

Comment:
Can you use `pymongo.ASCENDING` or `pymongo.DESCENDING` here? They are alias for `1` or `-1` (I forget which is which) and it easier is to understand what the code does.